src/components/Navbar.tsx

```diff
@@ -8,13 +13,16 @@ export const Navbar = () => {
     >
       <div className="container">
         <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
+          <a
+            className={`navbar-item ${path.includes('home') || path === '/' ? 'has-background-grey-lighter' : ''}`}
```

Comment:
```suggestion
            className={`navbar-item ${path.includes('home') || path === '/' ? 'has-background-grey-lighter' : ''}`}            
```

We can extract it to a variable (or a function) to avoid complexity in the JSX =)

```tsx
    const getHomeLinkClassNames = () => {
        const isHome = path.includes('home') || path === '/'

        return isHome ? 'has-background-grey-lighter' : ''
    }

    ...

    <a className={getHomeLinkClassNames()}>
```